platform/linuxbsd/detect.py

```diff
@@ -89,7 +90,7 @@ def get_flags():
     return []
 
 
-def configure(env):
+def configure(env, cc_version_int):
```

Comment:
Since you have added a new argument to the `configure`, the same change should be done for all other platforms.

Reply:
I'm not fond of adding this to the prototype. You can just get this info from methods.py where needed.

Reply:
> I'm not fond of adding this to the prototype. You can just get this info from methods.py where needed.

Thanks for pointing this out, I've pushed a fix to this.